app/styles/components/_rui-error-template.sass

```diff
@@ -18,6 +18,9 @@
 .rui-error-template-tagline
   font-size: $font-size-lg
 
+.rui-error-button
```

Comment:
Likely shouldn't overide the base styles like this.  This is how we get non-uniform one-offs.